server/src/main/java/io/littlehorse/server/KafkaStreamsServerImpl.java

```diff
@@ -357,6 +360,21 @@ public void getTaskDef(TaskDefId req, StreamObserver<TaskDef> ctx) {
         }
     }
 
+    @Override
+    @Authorize(resources = ACLResource.ACL_TASK_WORKER_GROUP, actions = ACLAction.READ)
+    public void getTaskWorkerGroup(TaskDefId taskDefIdPb, StreamObserver<TaskWorkerGroup> ctx) {
+        TaskDefIdModel taskDefId = TaskDefIdModel.fromProto(taskDefIdPb, TaskDefIdModel.class, requestContext());
+        TaskWorkerGroupModel taskWorkerGroup = internalComms.getObject(
+                new TaskWorkerGroupIdModel(taskDefId), TaskWorkerGroupModel.class, requestContext());
+        if (taskWorkerGroup == null) {
+            ctx.onError(
+                    new LHApiException(Status.NOT_FOUND, "Couldn't find TaskDef %s".formatted(taskDefIdPb.getName())));
```

Comment:
Shouldn't be `TaskDef`...it is possible that the `TaskDef` exists but the `TaskWorkerGroup` does not exist. For example, if there are no workers polling it.